java/src/jmri/jmrit/beantable/BeanTableDataModel.java

```diff
@@ -453,8 +445,8 @@ public void setValueAt(Object value, int row, int col) {
     }
 
     protected void deleteBean(int row, int col) {
-        DeleteBeanWorker worker = new DeleteBeanWorker(getBySystemName(sysNameList.get(row)));
-        worker.execute();
+        jmri.util.ThreadingUtil.runOnGUI(() ->
+            new DeleteBeanWorker(getBySystemName(sysNameList.get(row))));
```

Comment:
This change omit's the `.execute()` call. Is that intended?

Reply:
Yep, no longer required, though the Exception handling had been removed, restored in latest upload.